core/jazz_ui/src/styles.css

```diff
@@ -849,167 +851,167 @@ html.jazz {
 
 .icon-icon-AWS .path1:before {
     content: "\ea67";
-    color: rgb(17, 16, 14);
+   /* color: rgb(17, 16, 14);*/
```

Comment:
Why are these commented out?

Reply:
@oscarbrooksdev9 question
![image](https://user-images.githubusercontent.com/3027534/54794019-a5fd5780-4c02-11e9-99c1-d24db3b241a9.png)

Reply:
The way AWS logo is composed is very complicated and not intuitive. the code that is commented out is left there if anyone ever wanted to revert to the old behavior of the aws logo. I can remove this if you think it's not necessary.